docs/crio.8.md

```diff
@@ -25,6 +25,7 @@ crio
 [--log=[value]]
 [--log-format value]
 [--log-level value]
+[--log-journald]
```

Comment:
Docker Daemon calls this the log-driver  I would go with that, and jsut have kubernetes, journald as the two values.
```
--log-driver="json-file|syslog|journald|gelf|fluentd|awslogs|splunk|etwlogs|gcplogs|none"
         Default driver for container logs. Default is json-file.
         Warning: docker logs command works only for json-file logging driver.
```

Reply:
@rhatdan the language in the card: https://jira.coreos.com/browse/RUN-258 says "as well as". I'm not sure the exact use case, but it could be that someone wants it logged in both. Further, there are a lot of assumptions in conmon that there is a log-path to write to as well. Originally, my PR had them exclusive, but in the interest of simplicity and requirement fulfilment, went for this.
WDYT?

Reply:
OK Works for me. If this is what @mrunalp wants.

Reply:
@rhatdan @haircommander For CRI-O, we always need to write to the k8s log file. However, that may not be true for podman. So, I think we can go with your suggestion of using --log-driver and make it accept multiple options. For CRI-O, we will always pass k8s CRI option. So, something like `--log-driver=cri-file --log-driver=journal` or `--log-driver=cri-file,journal`. For podman, it could be just `--log-driver=journal` or any other drivers that we choose to support.

Reply:
Roger roger I'll look into it

Reply:
I think the best would be a single `--log-path` option, as:

 `--log-path=journal:`
`--log-path=k8s:/path/to/the/file`
`--log-path=/path/to/the/file` (default to k8s)

so if in the future we want to add more logging drivers we can simply specify a new prefix, and at the same time keep the ability to log to multiple destinations.